src/CookiePolicyBanner/_cookie-policy-banner.scss

```diff
@@ -18,12 +17,6 @@ $small-font-size: 14px;
   line-height: 1.5;
   position: relative;
 
-  font: {
-    family: $font-family-sans-serif !important;
-    size: 1rem;
-    weight: 400;
```

Comment:
[question] Should we delete these rules?

Reply:
Yes. These rules are used below in the main styles file